hacker_Rank/June1_Answer.py

```diff
@@ -0,0 +1,23 @@
+def solution(a):
+    a = list(a)
+    n = len(a)
+    try:
+        stock_price = max(
+            [[a[sell] - a[buy]] for buy in range(n) for sell in range(n) if 0 <= buy < sell if (a[sell] - a[buy]) > 0])
+
+
+
+    except ValueError:
+        yield 0
+    else:
+        yield stock_price
```

Comment:
``` yield ``` is used only when you need Lazy load, means you want generate Series of Item, 
in this case you get single Number as out put